content/en/core/overview/supported-software.md

```diff
@@ -9,7 +9,7 @@ relevantLinks: >
 
 Medic supports minor versions of the CHT Core Framework for three months after the next minor version is made available, and the latest minor of a major version for twelve months after the next major version is made available.
 
-Once a version is no longer supported it will not receive any further releases and upgrading may be required before Medic can investigate any issues.
+Once a version is no longer supported it will not receive any further patch releases.
```

Comment:
```suggestion
Once a version is no longer supported it will not receive any further patch releases and upgrading to a supported version will be required to resolve any issues you have.
```